app/models/conversation.rb

```diff
@@ -168,6 +168,14 @@ def username
     end
   end
 
+  def status_str
+    self.status.gsub('_', ' ').titleize
+  end
+
+  def lifecycle_str
+    self.lifecycle.gsub('_', ' ').titleize
```

Comment:
Lots of the app is internationalized. Would it make sense to use internationalized text instead of `.titleize`?

Reply:
It's a good question, but tbh I think internationalizing it was premature optimization in the first place. I'm only aware of a single instance (there were probably more) when the Spanish version was used. If we need to support additional languages down the road we can loop back.